share/functions/__fish_print_zypp_repos.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_print_zypp_repos -d "Print repositories of zypper package manager"
```

Comment:
>I noticed that pacman has a standalone function __fish_print_pacman_repos, so I thought I should put that into another file...

That's also used in other completions (pacman has a bit of a culture of things wrapping it). This one isn't.

>Is it a good idea to write a function __fish_print_repos and do the distribution detection inside it? Just like __fish_print_packages?

Please no. I don't like that one as it is, because any completion that uses it wants exactly one of the options. That means the detection is actually unnecessary, and can even be wrong (e.g. if you have `apt-cache` installed and try to use `pacman`, you'll get the wrong packages).